garak/resources/tap/tap_main.py

```diff
@@ -35,11 +35,7 @@
 SAVE_RESULTS = True
 
 resources_tap_data_file = (
-    garak._config.transient.cache_dir
-    / "resources"
-    / "tap"
-    / "data"
-    / "tap_jailbreaks.txt"
+    garak._config.transient.cache_dir / "data" / "tap" / "tap_jailbreaks.txt"
```

Comment:
Since `probes.tap` references the `prompts_location` as `data_path / "tap" / "tap_jailbreaks.txt"`, do we want this to reflect that as well? The idea is to keep your TAP results such that you can use them (or share them with us!) in subsequent probes. Though perhaps that's a larger issue.

Reply:
I agree these should be captured better for re-use. I will defer that change to a future iteration though. I think there is a larger issue to address as we have a few techniques that generate these intermediate files that could be valuable from a research context.